src/layouts/AppHeader.tsx

```diff
@@ -83,91 +83,121 @@ export function AppHeader() {
+            justifyContent: 'center',
+            gap: 2,
+            backgroundColor: 'black',
           }}
-          onClick={() => setMobileMenuOpen(false)}
         >
-          <img src={uiConfig.appLogo} alt="An SVG of an eye" height={20} />
+          <Typography sx={{ textAlign: 'center', color: 'white' }} variant="caption">
+            <Trans>
+              As per the community vote, ETH borrowing on the Ethereum Market will be paused ahead
+              of the merge.
+            </Trans>
+          </Typography>
+          <Button
+            component={Link}
+            variant="surface"
+            size="small"
+            href="https://snapshot.org/#/aave.eth/proposal/0xa121311c67b7a5bbe5b8b5fe1911663a0ab94ed339a6a4b0e1b9443f670a0e97"
+          >
+            <Trans>Learn more</Trans>
+          </Button>
         </Box>
-        <Box sx={{ mr: sm ? 1 : 3 }}>
-          {ENABLE_TESTNET && (
-            <ContentWithTooltip tooltipContent={testnetTooltip} offset={[0, -4]} withoutHover>
-              <Button
-                variant="surface"
-                size="small"
-                color="primary"
-                sx={{
-                  backgroundColor: '#B6509E',
-                  '&:hover, &.Mui-focusVisible': { backgroundColor: 'rgba(182, 80, 158, 0.7)' },
-                }}
-              >
-                TESTNET
-                <SvgIcon sx={{ marginLeft: '2px', fontSize: '16px' }}>
-                  <InformationCircleIcon />
-                </SvgIcon>
-              </Button>
-            </ContentWithTooltip>
-          )}
-        </Box>
-
-        <Box sx={{ display: { xs: 'none', md: 'block' } }}>
-          <NavItems />
-        </Box>
-
-        <Box sx={{ flexGrow: 1 }} />
+      )}
+      <HideOnScroll>
+        <Box
+          component="header"
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          sx={(theme) => ({
+            height: headerHeight,
+            position: 'sticky',
+            top: 0,
+            transition: theme.transitions.create('top'),
+            zIndex: theme.zIndex.appBar,
+            bgcolor: 'background.header',
+            padding: {
+              xs: mobileMenuOpen || walletWidgetOpen ? '8px 20px' : '8px 8px 8px 20px',
+              xsm: '8px 20px',
+            },
+            display: 'flex',
+            alignItems: 'center',
+            flexDirection: 'space-between',
+            boxShadow: 'inset 0px -1px 0px rgba(242, 243, 247, 0.16)',
+          })}
+        >
+          <Box
+            component={Link}
+            href="/"
+            aria-label="Go to homepage"
+            sx={{
+              lineHeight: 0,
+              mr: 3,
+              transition: '0.3s ease all',
+              '&:hover': { opacity: 0.7 },
+            }}
+            onClick={() => setMobileMenuOpen(false)}
+          >
+            <img src={uiConfig.appLogo} alt="An SVG of an eye" height={20} />
```

Comment:
Could we use nextjs's `<Image />` component here?

Reply:
This diff isn't very helpful, but this is existing. My changes are from lines 86 - 113